docs/reference-guides/block-api/block-transforms.md

```diff
@@ -223,18 +223,9 @@ transforms: {
 
 <h4 id="schemas-and-content-models">Schemas and Content Models</h4>
 
-When pasting content it's possible to define
-a [content model](https://html.spec.whatwg.org/multipage/dom.html#content-models) that will be used to validate and
-process pasted content. It's often the case that HTML pasted into the editor will contain a mixture of elements that _
-should_ transfer as well as elements that _shouldn't_. For example, consider
-pasting `<span class="time">12:04 pm</span>` into the editor. We want to copy `12:04 pm` and omit the `<span>` and
-its `class` attribute because those won't carry the same meaning or structure as they originally did from where they
-were copied.
-
-When writing `raw` transforms you can control this by supplying a `schema` which describes allowable content and which
-will be applied to clean up the pasted content before attempting to match with your block. The schemas are passed
-into [`cleanNodeList` from `@wordpress/dom`](https://github.com/wordpress/gutenberg/blob/trunk/packages/dom/src/dom/clean-node-list.js); check there for
-a [complete description of the schema](https://github.com/wordpress/gutenberg/blob/trunk/packages/dom/src/phrasing-content.js).
+When pasting content it's possible to define a [content model](https://html.spec.whatwg.org/multipage/dom.html#content-models) that will be used to validate and process pasted content. It's often the case that HTML pasted into the editor will contain a mixture of elements that _ should_ transfer as well as elements that _shouldn't_. For example, consider pasting `<span class="time">12:04 pm</span>` into the editor. We want to copy `12:04 pm` and omit the `<span>` and its `class` attribute because those won't carry the same meaning or structure as they originally did from where they were copied.
```

Comment:
thanks for the fix and sorry for the delay: ; looks great and ready to be merged except could you fix the following: `_should_` ; (no space between `_` and `should`)

Reply:
Thanks for the review 🙌🏻 I have removed the space between _ and should in d7bf196. 🙂